docs/reference/function/add_logger.txt

```diff
@@ -15,7 +15,7 @@ MongoDB\\add_logger()
 Definition
 ----------
 
-.. phpmethod:: MongoDB\\add_logger()
+.. phpmethod:: MongoDB\add_logger()
```

Comment:
@norareidy: The "Behavior" section later in this file has:

```rst
`Psr\\Log\\LoggerInterface <https://www.php-fig.org/psr/psr-3/#3-psrlogloggerinterface>`__
```

I assume the escaping there needs to stay since it's basic link syntax? We only need to remove escaping within directives.

Similar to what you had to do in https://github.com/mongodb/mongo-php-library/pull/1244/files#diff-0494d8195d629119e6de652538a76bd23063d2234cf08cad50eeebc94e6dee99R34

Reply:
Yeah, the `\\` should stay in that case since it's not a directive